### DISCUSSION TITLE
No Publisher classes detected in Publishers across all namespaces.

### DISCUSSION THREAD

**Discussion by l-vanel** [codeigniter4/shield#418] [Q&A]
-I want to publish the uploads directory, so i wrote this :

```php
namespace App\Publishers;
use CodeIgniter\Publisher\Publisher;

class UploadPublisher extends Publisher
{
    protected $source = WRITEPATH . 'uploads/';
    protected $destination = FCPATH . 'uploads';

    public function publish(): bool
    {
        return $this
            ->addPath('*')
            ->copy();
    }
}
```

-But i have this issue after doing php spark publish

No Publisher classes detected in Publishers across all namespaces.

**Comment by kenjis** [comment]
Here is Discussion for CodeIgniter Shield.
What's the relation?

If you need support for how to use Publisher, please go to the forum.
http://forum.codeigniter.com/forumdisplay.php?fid=30

**Comment by l-vanel** [comment]
I confused, sorry
